settings/typescript-language-server.vim

```diff
@@ -1,3 +1,38 @@
+function! s:find_vue_plugin() abort
+  let package_json_path = lsp#utils#find_nearest_parent_file(lsp#utils#get_buffer_path(), 'package.json')
```

Comment:
[ask]
What happens if there is vue in the root package.json In monorepo?
It would be nice if users could inject the location of package.json(global variable or can get from `lsp_settings#get()?`

Reply:
Thank you for your feedback! I couldn't consider that… 
What I want to do is enabling `@vue/typescript-plugin` only in Vue projects, but I don't come up with an idea to detect Vue projects even in monorepo situation 🤔 
Now I'm wondering if `@vue/typescript-plugin` is always enabled when it exists, since it doesn't have bad side effects (might have small performance degradation, though).

Reply:
I modified to use `@vue/typescript-plugin` when it just exists https://github.com/mattn/vim-lsp-settings/pull/728/commits/983dc1666b645451312f026307d345e79a4f9b97 .

I think that this still has small side effect to all users since only Vue users will install `volar-server`.